### ISSUE TITLE
Regarding the loss function

### ISSUE THREAD

**Issue by leviszhang** [altosaar/variational-autoencoder#28]
Hi, thank you for the nice implementation. Are there any references to the loss function "log_p_x_and_z - log_q_z" used in the Pytorch implementation? Both the two logarithm terms refer to a "NormalLogProb" class. It seems quite different from the one in the paper "Tutorial: Deriving the Standard Variational Autoencoder (VAE) Loss Function". Thank you for your time reading this.

**Comment by jaanaltosaar** [comment]
Yes! I derive it here: https://jaan.io/what-is-variational-autoencoder-vae-tutorial/ - feel free to follow up if you have questions :)
